command/server/config/config.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	LogFilePath              string     `json:"log_to" yaml:"log_to"`
 	JSONRPCBatchRequestLimit uint64     `json:"json_rpc_batch_request_limit" yaml:"json_rpc_batch_request_limit"`
 	JSONRPCBlockRangeLimit   uint64     `json:"json_rpc_block_range_limit" yaml:"json_rpc_block_range_limit"`
+	VerifyOutcomeApi         string     `json:"verify_outcome_api" yaml:"verify_outcome_api"`
```

Comment:
maybe have this field be `verify_outcome_api_url` (and VerifyOutcomeApiUrl) just so it's clearer

Reply:
actually, i would include this within the `data_feed` object too just so it's organized with the other datafeed-related fields